tests/sanity/ignore-2.9.txt

```diff
@@ -183,5 +183,9 @@ plugins/modules/azure_rm_adpassword_info.py validate-modules:nonexistent-paramet
 plugins/modules/azure_rm_adpassword_info.py validate-modules:import-before-documentation
 plugins/modules/azure_rm_adserviceprincipal.py validate-modules:nonexistent-parameter-documented
 plugins/modules/azure_rm_adserviceprincipal_info.py validate-modules:nonexistent-parameter-documented
+plugins/modules/azure_rm_backupazurevm.py validate-modules:required_if-requirements-unknown
+plugins/modules/azure_rm_backupazurevm.py validate-modules:required_if-unknown-key
+plugins/modules/azure_rm_backupazurevm_info.py validate-modules:required_if-requirements-unknown
+plugins/modules/azure_rm_backupazurevm_info.py validate-modules:required_if-unknown-key
```

Comment:
It's my mistake, Please delete 186,187,188 and 189 lines; Thank you very much!

Reply:
186-189 lines are removed from tests/sanity/ignore-2.9.txt